.ci/scripts/button-left-click.py

```diff
@@ -20,21 +20,22 @@ def button_left_click(device, start_x, start_y, offset_x=0, offset_y=0):
     device.emit(uinput.ABS_X, start_x, syn=False)
     device.emit(uinput.ABS_Y, start_y, syn=False)
     device.syn()
-    time.sleep(0.1)
+    time.sleep(0.2)
 
     # Press button.
     device.emit(uinput.BTN_LEFT, 1)
```

Comment:
What we want to stress is the touch interactions. We should to use BTN_TOUCH here:

```suggestion
    device.emit(uinput.BTN_TOUCH, 1)  # Press touch
```